python/nano/src/bigdl/nano/pytorch/inference/optimizer.py

```diff
@@ -92,105 +117,202 @@ def optimize(self, model,
+                                                                  accelerator=accelerator,
+                                                                  input_sample=input_sample,
+                                                                  # remove output of openvino
+                                                                  logging=False)
                     except Exception as e:
                         print(e)
+                        continue
 
                 # if precision is int8 or bf16, then we will use quantize method
-                if split_method[1] == "int8" or split_method[1] == "bf16":
-                    precision = split_method[1]
-                    q_method = None if len(split_method) < 4 else split_method[3]
+                elif precision in ("int8", "bf16"):
+                    ort_method = _detect_ort_method(method)
                     try:
-                        # TODO: remove the logging of quantization
-                        accelerated_model = Trainer.quantize(model=model,
+                        accelerated_model = Trainer.quantize(model=deepcopy(model),
                                                              precision=precision,
                                                              accelerator=accelerator,
                                                              use_ipex=use_ipex,
                                                              calib_dataloader=training_data,
-                                                             method=q_method)
-                        result_map[method] = {}
-
-                        def func_test(model, input_sample):
-                            model(*input_sample)
-                        result_map[method]["latency"] =\
-                            _throughput_calculate_helper(100, func_test,
-                                                         accelerated_model, input_sample)
-                        if validation_data is not None and metric is not None:
-                            result_map[method]["accuracy"] =\
-                                _accuracy_calculate_helper(accelerated_model,
-                                                           metric, validation_data)
-                        result_map[method]["model"] = accelerated_model
+                                                             method=ort_method,
+                                                             # remove output of openvino
+                                                             logging=False)
                     except Exception as e:
                         print(e)
+                        continue
+
+                result_map[method] = {}
+
+                def func_test(model, input_sample):
+                    # with torch.cpu.amp.autocast():
+                    model(*input_sample)
+
+                torch.set_num_threads(cpu_num)
```

Comment:
This setting is only useful for `accelerator=None`

Reply:
For onnxruntime, you need to set onnxrt session opt during the trianing.
For openvino, we don't have this setting right now, we need to find a way to set this.

Reply:
Done